application/docs/methodology/ci-cd/github/02-project-structure.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 2
+---
+
+# Project Structure
+
+Set up your repository structure for CI/CD following Hoverkraft conventions.
+
+## Directory Layout
+
+Standard structure for any web application:
+
+```
+your-project/
+├── .github/
+│   ├── dependabot.yml
+│   └── workflows/
+│       ├── __shared-ci.yml
+│       ├── main-ci.yml
+│       ├── pull-request-ci.yml
+│       └── (additional workflows...)
+├── src/                    # Your application code
+│   └── ...
+├── Dockerfile             # Optional: for containerized apps
+├── Makefile              # Build automation
+├── README.md
+├── CONTRIBUTING.md
+└── LICENSE
+```
+
+**Alternative structure** (monorepo or specific frameworks):
+
+```
+your-project/
+├── .github/workflows/
+├── application/           # Or: app/, packages/, etc.
```

Comment:
In any case we always have an application folder this is the standard, in this folder we find the application(s) code